var/spack/repos/builtin/packages/py-cx-oracle/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCxOracle(PythonPackage):
+    """Python interface to Oracle"""
+
+    homepage = "https://oracle.github.io/python-cx_Oracle"
+    pypi     = "cx_Oracle/cx_Oracle-8.3.0.tar.gz"
+
+    version('8.3.0', sha256='3b2d215af4441463c97ea469b9cc307460739f89fdfa8ea222ea3518f1a424d9')
+
+    depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('oracle-instant-client', type=('build', 'link', 'run'))
```

Comment:
Where do you see this dep listed?

Reply:
It is needed at runtime at least:

> Add Oracle 21, 19, 18, 12 or 11.2 client libraries to your operating system library search path such as PATH on Windows or LD_LIBRARY_PATH on Linux

(from [here](https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html#quick-start-cx-oracle-installation))

Reply:
But looks like it's not needed at build-time